core/src/test/java/org/testcontainers/containers/ReusabilityUnitTests.java

```diff
@@ -61,8 +67,8 @@
         @Parameterized.Parameters(name = "{0}")
         public static Object[][] data() {
             return new Object[][] {
-                { "generic", new GenericContainer(IMAGE_FUTURE), true },
-                { "anonymous generic", new GenericContainer(IMAGE_FUTURE) {}, true },
+                { "generic", new GenericContainer<>(new RemoteDockerImage(IMAGE_FUTURE)), true },
```

Comment:
FTR if I remember correctly, I had to use `IMAGE_FUTURE` here to avoid the side effect we had in the constructor, but now it can be as simple as `new GenericContainer<>(TINY_IMAGE)`

Reply:
Updated